dss-api.yml

```
@@ -1385,7 +1385,7 @@ paths:
                     name:
                       description: Name of the file.
                       type: string
-                      pattern: "^[^/\\\\]+$"
+                      pattern: "(?=^((?!\\.\\/).)*$)(?=^((?!\\/\\.).)*$)(?=^(?!\\/).*$)(?=^(?!~\\/).*$)(?=^(?!\\.\\.$).*)(?=^(?!\\.$).*)"
```

I suggest this simplified regex
```suggestion
                      pattern: "^(\\.?[\\w\\d][\\w\\d\\s.]*)+((\\.?[\\w\\d][\\w\\d\\s.]*)|\\/)*$"
```

---

Sure, sounds good.